mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -523,10 +588,13 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD s isPartOf (Laij;)Z
 		ARG 1 entity
 	METHOD t isSpectator ()Z
-	METHOD t handlePlayerAttack (Laij;)Z
+	METHOD t onAttack (Laij;)Z
```

Comment:
This is a weird one. It is only called by players, has a special behaviour only when the attacker is a player, but still accepts an Entity. And it is used as such:
```java
   public void attack(Entity entity_1) {
      if (entity_1.canPlayerAttack()) {
         if (!entity_1.handlePlayerAttack(this)) {
```
So `handleAttack` makes a lot more sense than `onAttack`. Now, whether we keep `player` in the name is up to debate.